src/stylesheets/components/_notFound.scss

```diff
@@ -1,3 +1,14 @@
 .not-found-page {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    padding-bottom: $header-height;
```

Comment:
This change causes the text to be centered, but the page still scrolls. Remove the bottom padding and make the height `calc(100% - #{$header-height})` to get the desired result